DigiMeSDK/Postbox/Classes/DMEAPIClient+Postbox.h

```diff
@@ -0,0 +1,32 @@
+//
+//  DMEAPIClient+Postbox.h
+//  DigiMeSDK
+//
+//  Created on 23/05/2019.
+//  Copyright © 2019 digi.me Limited. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "DMEClientConfiguration.h"
+#import "DMEAPIClient.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface DMEAPIClient (Postbox)
+
+/**
+ Pushes data to user's Postbox.
+ 
+ @param postbox CAPostbox
+ @param metadata NSData
+ @param data NSData
+ @param completion PostboxDataPushCompletionBlock
+ */
+- (void)pushDataToPostboxWithPostbox:(CAPostbox *)postbox
+                      metadataToPush:(NSData *)metadata
```

Comment:
Andt these without the `ToPush` on the end. Instead we can add these to the method descrition, ie:

```objective-c
/**
 Pushes data to user's Postbox.
 
 @param postbox CAPostbox - where data is to be pushed
 @param metadata NSData - to push to given Postbox
 @param data NSData - to push to given Postbox
 @param completion PostboxPushCompletionBlock
 */
- (void)pushDataToPostbox:(CAPostbox *)postbox
                      metadata:(NSData *)metadata
                          data:(NSData *)data
                          completion:(void(^)(NSError * _Nullable error))completion;

```

Also note - the method description says it takes `PostboxDataPushCompletionBlock` as completion, but the method signature says otherwise - which should it be?